packages/api/app/Models/Permission.js

```diff
@@ -94,6 +95,13 @@ class Permission extends Model {
 				return false;
 			}
 		}
+		/** Individual Reviewer Permissions */
+		if (matchesPermission([permissions.CREATE_TECHNOLOGY_REVISION], matchedPermission)) {
+			const technologyReviewed = await Technology.getTechnology(technology);
+			const reviewer = await Reviewer.getReviewer(user);
+			const isReviewer = await reviewer.isReviewer(technologyReviewed);
+			if (!isReviewer) return false;
```

Comment:
```suggestion
			return reviewer.isReviewer(technologyReviewed)
```

acho que novamente não precisa desse await aqui